prisma/migrations/20220831141824_transaction_block_height/migration.sql

```diff
@@ -0,0 +1,8 @@
+/*
+  Warnings:
+
+  - Added the required column `blockHeight` to the `Transaction` table without a default value. This is not possible if the table is not empty.
```

Comment:
I'm not sure how much this matters as the branch seems to work fine to me, but it looks like it is possible to add a column with a default value even if the table has data

https://stackoverflow.com/questions/92082/add-a-column-with-a-default-value-to-an-existing-table-in-sql-server